### ISSUE TITLE
Pass existing gaussian splatting checkpoint into prune script

### ISSUE THREAD

**Issue by alpergel** [VITA-Group/LightGaussian#7]
Hello, how do we provide the argument to pass our existing gaussian splatting checkpoint in ply to the pruning script?

**Comment by Kevin-2017** [comment]
Hello @alpergel, thank you so much for taking the time to look over our work! As of now, our script is designed to work with a PyTorch model file (xxxx.pth), as it includes all the crucial gradients and iterations information. Considering it might be a important need we would add support for ply potentially by tomorrow. However, please note that this might require some tweaking of the hyperparameters. I'll updated you when this part is realy!

**Comment by alpergel** [comment]
Hello, great thank you so much!

**Comment by Kevin-2017** [comment]
:D

**Comment by Kevin-2017** [comment]
You can now use --start_pointcloud path/to/scene.plt with prune_finetune.py.  Just a couple of helpful tips for adjusting the hyperparameters during tuning:

1. Keep in mind that the iterations will begin from 0. So, remember to adjust the prune iteration to 1 and modify other related hyperparameters accordingly.
2. It's a good idea to reduce the learning rate for the xyz values, considering that the original training utilized an exponential scheduler.

 If you have any questions or need further assistance, feel free to reopen and ask!

**Comment by alpergel** [comment]
Hello, thank you so much! Can you give me your suggested hyperparameters that I can get started with in the form of a command line input? Thank you so much!
